api/rbac/v1alpha1/role_types.go

```diff
@@ -11,9 +11,7 @@ type Role struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	KargoManaged      bool                `json:"kargoManaged,omitempty" protobuf:"varint,2,opt,name=kargoManaged"`
-	Subs              []string            `json:"subs,omitempty" protobuf:"bytes,3,rep,name=subs"`
-	Emails            []string            `json:"emails,omitempty" protobuf:"bytes,4,rep,name=emails"`
-	Groups            []string            `json:"groups,omitempty" protobuf:"bytes,5,rep,name=groups"`
+	Claims            []Claim             `json:"claims,omitempty" protobuf:"bytes,3,rep,name=claims"`
```

Comment:
```suggestion
	Claims            []Claim             `json:"claims,omitempty" protobuf:"bytes,7,rep,name=claims"`
```

Reply:
I'm not completely convinced what the number means so thank you for pointing that out.